src/components/point-light.js

```diff
@@ -51,10 +59,15 @@ AFRAME.registerComponent("point-light", {
     if (width !== prevWidth || height !== prevHeight) {
       light.shadow.mapSize.set(width, height);
 
-      if (light.shadow.map) {
-        light.shadow.map.dispose();
-        light.shadow.map = null;
-      }
+      // if (light.shadow.map) {
```

Comment:
Any reason to keep this commented out code?

Reply:
Hmm. Maybe I should uncomment that? I'll test.